v2.3_release_notes.md

```diff
@@ -0,0 +1,88 @@
+Intel® Neural Compressor v2.3 Release
+---
+
+- Highlights
+- Features
+- Improvement
+- Productivity
+- Bug Fixes
+- Examples
+- Validated Configurations
+
+
+**Highlights**
```

Comment:
Add Sparsity (sparseGPT) as 3rd highlight

Reply:
Add INC integration into ONNX Runtime and Olive as the 1st highlight

Reply:
updated